bin/bossSwitch.py

```diff
@@ -126,21 +155,14 @@ def load_obj(name):
         Returns:
             object saved within .pkl file
     """
-    with open('../' + name + '.pkl', 'rb') as f:
+    with open(real_path + '/' + name + '.pkl', 'rb') as f:
         return pickle.load(f)
 
-class bcolors:
-    """
-        Used to add coloring to terminal print statements
-    """
-    OKBLUE = '\033[94m'
-    OKGREEN = '\033[92m'
-    WARNING = '\033[93m'
-    FAIL = '\033[91m'
-    ENDC = '\033[0m'
-
 if __name__ == '__main__':
 
+    #Grab files path to use as reference.
+    real_path = constants.repo_path()
```

Comment:
You're using `real_path` like a global variable in the rest of the script, but it's only a local variable.

Reply:
real_path is a global variable. Maybe I should define it in capital letters as per custom? But it is global

Reply:
The code is functional using real_path as a global variable as is now

Reply:
Yep, you're right. I never thought about all the vars defined in the `if __name__ == '__main__':` block as being global.